kolibri/plugins/coach/assets/src/views/common/notifications/__test__/ActivityList.spec.js

```diff
@@ -197,12 +197,6 @@ describe('ActivityList component', () => {
     });
     const { wrapper } = makeWrapper({});
 
-    wrapper.setMethods({
```

Comment:
This seems to be unnecessary as tests pass without it and I haven't found a place in tests and code where it would be needed. I just wanted to highlight it just in case I've missed something.